JitPackでのビルドが失敗している

https://jitpack.io/#kusumotolab/kGenProg
https://jitpack.io/com/github/kusumotolab/kGenProg/v1.8.1/build.log
```
> Task :compileJava
/home/jitpack/build/src/main/java/jp/kusumotolab/kgenprog/KGenProgMain.java:297: error: cannot find symbol
      return max.isEmpty() ? "--" : format.format(max.orElse(Double.NaN));
                ^
  symbol:   method isEmpty()
  location: variable max of type OptionalDouble
/home/jitpack/build/src/main/java/jp/kusumotolab/kgenprog/KGenProgMain.java:306: error: cannot find symbol
      return min.isEmpty() ? "--" : format.format(min.orElse(Double.NaN));
                ^
  symbol:   method isEmpty()
  location: variable min of type OptionalDouble
/home/jitpack/build/src/main/java/jp/kusumotolab/kgenprog/project/jdt/JDTASTConstruction.java:126: error: cannot find symbol
      return Files.readString(Paths.get(path), charset);
                  ^
  symbol:   method readString(Path,Charset)
  location: class Files
/home/jitpack/build/src/main/java/jp/kusumotolab/kgenprog/output/PatchGenerator.java:54: error: cannot find symbol
    return List.of(source.split(delimiter));
               ^
  symbol:   method of(String[])
  location: interface List
4 errors

> Task :compileJava FAILED

FAILURE: Build failed with an exception.
```

JitPackがJDK8を使ってることが原因．
https://jitpack.io/docs/BUILDING/#java-version

>JitPack will compile projects using OpenJDK Java 8. See the example projects on how to set a different target version in your build file.
>Maven projects that specify a target version in their pom will be built with that target version.
>If your project uses Travis or Circle CI then JitPack will read the lowest jdk version from yml file and use that to build.
>Alternatively create a jitpack.yml file in the root of your repository and specify a jdk version

--- a/build.gradle
+++ b/build.gradle
@@ -17,7 +17,7 @@ plugins {
     id 'jacoco'
 
     // Maven plugin to use JitPack
-    id 'maven-publish'
+    id 'maven'
 
     // Gradle lint
     id "nebula.lint" version "16.9.0"

--- a/jitpack.yml
+++ b/jitpack.yml
@@ -1,0 +1,2 @@
+jdk:
+  - openjdk11

